docs/_docs/02_features/hooks.md

```diff
@@ -58,3 +58,19 @@ to Terraform.
 
 You can learn more about all the various configuration options supported in [the reference docs for the terraform
 block](/docs/reference/config-blocks-and-attributes/#terraform).
+
+## Error Hooks
+Think of error hooks like an exception handler, they allow you to specifiy a list of expressions that can be used to catch errors and run custom commands when that occur.
+
+Here is an example:
+``` hcl
+terraform {
+  error_hook "import_resource" {
```

Comment:
I think `error_hook` should be referenced in the "Blocks" list like `after_hook` / `before_hook`, otherwise, users will not know that such block can be used in HCL

https://terragrunt.gruntwork.io/docs/reference/config-blocks-and-attributes/

https://github.com/gruntwork-io/terragrunt/blob/master/docs/_docs/04_reference/config-blocks-and-attributes.md

Reply:
good point, let me update that

Reply:
updated https://github.com/gruntwork-io/terragrunt/pull/1967/commits/a467f8c6f0ec3f49d9ebf8f6b4800502df71702b